app/views/shared/dwp/_online.html.slim

```diff
@@ -0,0 +1,3 @@
+.row.pad-top-thirty-px
```

Comment:
I agree, this was a first pass at the problem to alert the users (and us, via the healthcheck) that the status had changed.  My long term plan has a store for the time of state changes so that we could turn off the green bar after X hours.